kernel/libs/aster-bigtcp/src/errors.rs

```diff
@@ -10,7 +10,41 @@ pub enum BindError {
 }
 
 pub mod tcp {
-    pub use smoltcp::socket::tcp::{ConnectError, ListenError, RecvError, SendError};
+    pub use smoltcp::socket::tcp::{RecvError, SendError};
+
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord)]
+    pub enum ListenError {
+        InvalidState,
+        Unaddressable,
+        /// The specified address is in use.
+        InUse,
```

Comment:
`AddressInUse`? It's a bit longer, but still shorter than `Unaddressable`, so I guess it's slightly better.